README.md

```diff
@@ -344,7 +344,7 @@ You can also define multiple files in `config/samples`, but you must specify the
    - `beta`: (Optional) Determines whether ModuleTemplate must have the beta flag or not. The type is bool.
    - `labels`: (Optional) Additional labels for ModuleTemplate.
    - `annotations`: (Optional) Additional annotations for ModuleTemplate.
-   - `customStateCheck`: (Optional) Specifies custom state checks for the module.
+   - `customStateCheck`: (Optional) DEPRECATED, Specifies custom state checks for the module.
```

Comment:
Should we maybe include a date when it was deprecated? Not sure if strictly necessary, but shouldn't harm either.

Reply:
@mmitoraj What do you think?

Reply:
I agree